tutor/src/components/navbar/preview-add-course-btn.jsx

```diff
@@ -26,6 +26,12 @@ export default class PreviewAddCourseBtn extends React.PureComponent {
     tourContext: React.PropTypes.instanceOf(TourContext),
   }
 
+  constructor(props) {
+    super(props);
+
+    this.onAddCourse.bind(this);
+  }
+
   @computed get course() {
```

Comment:
This shouldn't be needed, `onAddCourse` is decorated with `@action.bound`

Reply:
i tried, and it errored without it, will check again

Reply:
you're right,weird, i thought i saw it erroring, but it works :)